### DISCUSSION TITLE
Refactor configuration

### DISCUSSION THREAD

**Discussion by danilshvalov** [nvim-orgmode/orgmode#535] [Ideas]
Hi. Thank you for this wonderful plugin! I would like to suggest refactoring the configuration. For example

```lua
require('orgmode').setup({
  org_todo_keywords = {'TODO', 'WAITING', '|', 'DONE', 'DELEGATED'},
  org_todo_keyword_faces = {
    WAITING = ':foreground blue :weight bold',
    DELEGATED = ':background #FFFFFF :slant italic :underline on',
    TODO = ':background #000000 :foreground red', -- overrides builtin color for `TODO` keyword
  }
})
```

we can change to
```lua
require("orgmode").setup({
  todo = {
    keywords = {
      { name = "TODO", faces = ":background #000000 :foreground red" },
      { name = "WAITING", faces = ":foreground blue :weight bold" },
      { name = "|" },
      { name = "DONE" },
      { name = "DELEGATED", faces = ":background #FFFFFF :slant italic :underline on" },
    },
  },
})
```

If you like this idea, I can help you implement it.

**Comment by kristijanhusak** [comment]
Hi, thank you for the idea, but we do not plan to implement it. We try to follow Emacs orgmode configuration variables as much as possible. In this example, emacs variable is `org-todo-keyword` while we use the one with underscore `org_todo_keyword`

**Comment by mcepl** [comment]
Moreover, there are some people (like me ;)), who rely on those keywords for their extensions (https://git.sr.ht/~mcepl/editorconfig-orgmode.nvim).
